src/components/Icon.tsx

```diff
@@ -49,6 +49,11 @@ export const Icon = React.memo((props: IconProps) => {
  */
 export const Icons = {
   // Actions
+  kanban: (
+    <>
+      <path d="M19.893 3.001H4C2.897 3.001 2 3.898 2 5.001V19.001C2 20.104 2.897 21.001 4 21.001H19.893C20.996 21.001 21.893 20.104 21.893 19.001V8V7V6V5.001V5C21.892 3.897 20.994 3.001 19.893 3.001ZM8 19.001H4V8H8V19.001ZM14 19.001H10V8H14V19.001ZM16 19.001V8H19.893L19.894 19.001H16Z" />
+    </>
```

Comment:
**nit:** I don't think we need the surrounding `<>...</>` element, but not a big deal.